src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs

```diff
@@ -67,16 +67,16 @@ protected override void OnSizeChanged(int w, int h, int oldw, int oldh)
 		{
 			base.OnSizeChanged(w, h, oldw, oldh);
 
-			canvasBitmap = Bitmap.CreateBitmap(w, h, Bitmap.Config.Argb8888);
+			canvasBitmap = Bitmap.CreateBitmap(w, h, Bitmap.Config.Argb8888!)!;
 			drawCanvas = new Canvas(canvasBitmap);
 			LoadPoints();
 		}
 
-		protected override void OnDraw(Canvas canvas)
+		protected override void OnDraw(Canvas? canvas)
 		{
 			base.OnDraw(canvas);
 
-			canvas.DrawBitmap(canvasBitmap, 0, 0, canvasPaint);
+			canvas!.DrawBitmap(canvasBitmap!, 0, 0, canvasPaint);
```

Comment:
if we receive a nullable canvas, should we trust that isn't null here? Does the base call check for null?